ui/src/Components/SearchCandidates.jsx

```diff
@@ -37,63 +28,82 @@ export default class SearchCandidates extends React.Component {
 		.catch((error) => {console.log(error)});
 	}
 
-	/** Event Handlers for search fields
-		// TODO: Uncomment so state is updated, triggering a rerender of results
 	onNameSearch = (event) => {
-		this.setState({ searchName: event.target.value });
+		this.setState({ searchName: event.target.value.toLowerCase() });
+		console.log(event.target.value)
 		};
 
 	onEmailSearch = (event) => {
-		this.setState({ searchEmail: event.target.value });
+		this.setState({ searchEmail: event.target.value.toLowerCase() });
 		};
 
 	onNumberSearch = (event) => {
 		this.setState({ searchNumber: event.target.value });
 		};
 
-	*/
+	populate = (a, obj, i)=> {
+		a.push(<tr>
+			<th scope="row">{i+1}</th>
+			<td ><a href={"/profiles/"+obj[i].profile_id}>{obj[i].first_name} {obj[i].last_name}</a></td>
+			<td>{obj[i].email}</td>
+			<td>{obj[i].contact_num}</td>
+		</tr>)
+	}
 
 	render(){
-		console.log("RENDERING");
-		// Add results row by row
 		if (this.state.search) {
-			const obj = this.state.search.result; // instance of cleaner data queried earlier
+			const obj = this.state.search.result
+			console.log(obj[0])
```

Comment:
is it best practice to comment out console logs in production?

Reply:
good call!